src/ttl/Writer.cpp

```diff
@@ -135,8 +135,16 @@ osm2rdf::ttl::Writer<T>::Writer(const osm2rdf::config::Config& config,
       generateIRI(osm2rdf::ttl::constants::NAMESPACE__OSM2RDF_MEMBER, "role");
   osm2rdf::ttl::constants::IRI__OSM2RDF_MEMBER__POS =
       generateIRI(osm2rdf::ttl::constants::NAMESPACE__OSM2RDF_MEMBER, "pos");
+  osm2rdf::ttl::constants::IRI__OSMMETA_CHANGESET =
+      generateIRI(osm2rdf::ttl::constants::NAMESPACE__OSM_META, "changeset");
   osm2rdf::ttl::constants::IRI__OSMMETA_TIMESTAMP =
       generateIRI(osm2rdf::ttl::constants::NAMESPACE__OSM_META, "timestamp");
+  osm2rdf::ttl::constants::IRI__OSMMETA_USER =
+      generateIRI(osm2rdf::ttl::constants::NAMESPACE__OSM_META, "user");
+  osm2rdf::ttl::constants::IRI__OSMMETA_VERSION =
+      generateIRI(osm2rdf::ttl::constants::NAMESPACE__OSM_META, "version");
+  osm2rdf::ttl::constants::IRI__OSMMETA_VISIBLE =
+      generateIRI(osm2rdf::ttl::constants::NAMESPACE__OSM_META, "visible");
```

Comment:
`visible` is [guaranteed](https://wiki.openstreetmap.org/wiki/Elements#Common_attributes) to be either `true` or `false` in the OSM XML format, so the corresponding triples could have Boolean values instead of strings. That said, `visible` will always be `true` in any normal extract of OSM or OHM data unless you specifically load an extract that contains full history.

Reply:
I agree, I changed it to boolean values. Also, the visibility triples are now only written if visibility is false.